sourcegraph/sourcegraph
# Insights/fix-sort-limit-change

Closes https://github.com/sourcegraph/sourcegraph/issues/36501

This is the "easy" version of the fix. We simply clear all of the selected series when changing sorting or limiting.

Maintaining the selections requires more backend work to support it.

## Test plan

- Go to any sortable insight with more than 1 series
- Toggle any series so that only 1 is shown
- Change the limit or sorting options
- Selections should be cleared showing all series as per the filters

## App preview:

- [Web](https://sg-web-insights-fix-sort-limit-change.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-pysyzykhod.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
